cluster/v1/types.go

```diff
@@ -257,3 +257,16 @@ const (
 	// relating to the ManagedCluster is removed when the ManagedCluster is deleted.
 	ManagedClusterFinalizer = "cluster.open-cluster-management.io/api-resource-cleanup"
 )
+
+const (
+	// ManagedClusterConditionDeleting is a condition which means the cluster is in deletion process.
+	ManagedClusterConditionDeleting string = "ManagedClusterIsDeleting"
```

Comment:
Deleting is good enough, it is ManagedCluster resource so it is not needed to specify it in the type

Reply:
done